plugins/templates/labhub/promotions/newcomers.jinja2.md

```diff
@@ -1,9 +1 @@
-Welcome @{{ target }}! :tada:
-
-We've just sent you an invite via email. Please accept it before proceeding forward.
```

Comment:
We still need an invite process via corobo, so this file and related code shouldnt be deleted.